### ISSUE TITLE
Pause shortcut as Mute if radio playing

### ISSUE THREAD

**Issue by Solarunit** [Taiko2k/TauonMusicBox#966]
I have a Pause global shortcut to play/pause local music in tauon, but I noticed some issues when using it when radio is playing.
So my idea is something like a new settings that does mute/unmute radio when you press play/pause shortcut.

**Comment by C0rn3j** [comment]
> I noticed some issues when using it when radio is playing.

Do you still have issues with the radio? What are they?

I think it'd be better to improve the radio support rather than having such a workaround, and you can somewhat already implement it yourself by convincing your OS sound manager to mute the application on some specific shortcut.

**Comment by Solarunit** [comment]
>Do you still have issues with the radio? What are they?

I have various issues with radios:

- sometime it doesn't reconnect after beeing unpaused 
- lost connection, can't reconnect, UI is frozen, UI is frozen when connecting to other radio or playing local file.
- unstable connection when nothing is paying for several seconds

>I think it'd be better to improve the radio support rather than having such a workaround, 

I agree


>and you can somewhat already implement it yourself by convincing your OS sound manager to mute the application on some specific shortcut.

I guess tauon already has mute shortcut, so I can use it instead of pause. But problem is you always have to remember that it's radio playing and not local file.

**Comment by C0rn3j** [comment]
I have very sporadically been able to repro the UI freezes, seems to happen every couple hours or not at all for a day.

Might be related to my network connections, as sometimes I am on a network with constant packet loss.

**Comment by Solarunit** [comment]
Yes, often happens when trying to connect/reconnect to some radio and selecting another station.
